vendor/assets/stylesheets/components/_roboto.scss

```diff
@@ -1,49 +1,49 @@
-    src: url("#{$roboto-font-path}Roboto-Light.eot?#iefix") format('embedded-opentype'),
-        url("#{$roboto-font-path}Roboto-Light.woff2") format("woff2"),
-        url("#{$roboto-font-path}Roboto-Light.woff") format("woff"),
-        url("#{$roboto-font-path}Roboto-Light.ttf") format("truetype");
+    src: local(Roboto Light), font-url('#{$roboto-font-path}Roboto-Light.eot');
+    src: font-url("#{$roboto-font-path}Roboto-Light.eot?#iefix") format('embedded-opentype'),
+        font-url("#{$roboto-font-path}Roboto-Light.woff2") format("woff2"),
+        font-url("#{$roboto-font-path}Roboto-Light.woff") format("woff"),
+        font-url("#{$roboto-font-path}Roboto-Light.ttf") format("truetype");
     font-weight: 300;
 }
 
 @font-face {
     font-family: "Roboto";
-    src: local(Roboto Regular), url('#{$roboto-font-path}Roboto-Regular.eot');
-    src: url("#{$roboto-font-path}Roboto-Regular.eot?#iefix") format('embedded-opentype'),
-        url("#{$roboto-font-path}Roboto-Regular.woff2") format("woff2"),
-        url("#{$roboto-font-path}Roboto-Regular.woff") format("woff"),
-        url("#{$roboto-font-path}Roboto-Regular.ttf") format("truetype");
+    src: local(Roboto Regular), font-url('#{$roboto-font-path}Roboto-Regular.eot');
+    src: font-url("#{$roboto-font-path}Roboto-Regular.eot?#iefix") format('embedded-opentype'),
+        font-url("#{$roboto-font-path}Roboto-Regular.woff2") format("woff2"),
+        font-url("#{$roboto-font-path}Roboto-Regular.woff") format("woff"),
+        font-url("#{$roboto-font-path}Roboto-Regular.ttf") format("truetype");
     font-weight: 400;
 }
 
 @font-face {
     font-family: "Roboto";
-    src: url('#{$roboto-font-path}Roboto-Medium.eot');
-    src: url("#{$roboto-font-path}Roboto-Medium.eot?#iefix") format('embedded-opentype'),
-        url("#{$roboto-font-path}Roboto-Medium.woff2") format("woff2"),
-        url("#{$roboto-font-path}Roboto-Medium.woff") format("woff"),
-        url("#{$roboto-font-path}Roboto-Medium.ttf") format("truetype");
+    src: font-url('#{$roboto-font-path}Roboto-Medium.eot');
+    src: font-url("#{$roboto-font-path}Roboto-Medium.eot?#iefix") format('embedded-opentype'),
+        font-url("#{$roboto-font-path}Roboto-Medium.woff2") format("woff2"),
+        font-url("#{$roboto-font-path}Roboto-Medium.woff") format("woff"),
+        font-url("#{$roboto-font-path}Roboto-Medium.ttf") format("truetype");
     font-weight: 500;
 }
 
 @font-face {
     font-family: "Roboto";
-    src: url('#{$roboto-font-path}Roboto-Bold.eot');
-    src: url("#{$roboto-font-path}Roboto-Bold.eot?#iefix") format('embedded-opentype'),
-        url("#{$roboto-font-path}Roboto-Bold.woff2") format("woff2"),
-        url("#{$roboto-font-path}Roboto-Bold.woff") format("woff"),
-        url("#{$roboto-font-path}Roboto-Bold.ttf") format("truetype");
+    src: font-url('#{$roboto-font-path}Roboto-Bold.eot');
+    src: font-url("#{$roboto-font-path}Roboto-Bold.eot?#iefix") format('embedded-opentype'),
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Prefer double-quoted strings